finch/processes/wps_xsubset_point_dataset.py

```diff
@@ -94,11 +92,16 @@ def _handler(self, request: WPSRequest, response: ExecuteResponse):
 
         variable = request.inputs["variable"][0].data
         variables = None if variable is None else [variable]
-        rcp = single_input_or_none(request.inputs, "rcp")
+        scenario = single_input_or_none(request.inputs, "scenario")
+        models = [m.data.strip() for m in request.inputs["models"]]
 
-        dataset_name = single_input_or_none(request.inputs, "dataset_name")
+        dataset_name = single_input_or_none(request.inputs, "dataset")
+        if dataset_name == 'bccaqv2':
+            dataset_name = 'candcs-u5'
+        dataset = get_datasets_config()[dataset_name]
```

Comment:
@aulemahal Failing to provide the "dataset" attribute in the request will result in a `KeyError: None`.

Reply:
Oh, I thought I had a default value! I'll make it so it defaults to the first dataset in the config (candcs-u5). (this will be documented in the process I believe)

Reply:
I instead chose to use the already existing `default_dataset` config value. Instead of being parsed in the process code, it is now used to fill the "default" field of the WPS input directly.